src/main/java/com/client/core/resttrigger/controller/jobsubmission/JobSubmissionRestTriggerController.java

```diff
@@ -17,67 +21,95 @@
 import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.bind.annotation.ResponseBody;
 
+import java.util.Arrays;
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
+import java.util.stream.Collectors;
 
 @Controller
 @RequestMapping("/resttrigger/jobsubmission/*")
 public class JobSubmissionRestTriggerController extends AbstractRestTriggerController<JobSubmission, JobSubmissionRestTriggerHelper, JobSubmissionRestTriggerTraverser> {
 
     private final Logger log = Logger.getLogger(JobSubmissionRestTriggerController.class);
+    private final JsonConverter jsonConverter;
 
     @Autowired
-    public JobSubmissionRestTriggerController(Optional<List<TriggerValidator<JobSubmission, JobSubmissionRestTriggerHelper, JobSubmissionRestTriggerTraverser>>> triggerValidators) {
+    public JobSubmissionRestTriggerController(Optional<List<TriggerValidator<JobSubmission, JobSubmissionRestTriggerHelper, JobSubmissionRestTriggerTraverser>>> triggerValidators, JsonConverter jsonConverter) {
         super(JobSubmission.class, triggerValidators, JobSubmissionRelatedEntity.values());
+        this.jsonConverter = AppContext.getApplicationContext().getBean(JsonConverter.class);
     }
 
     /**
      * Called when job is added.
      *
      * @param body
-     *
      * @return the json parsed form response message
      */
-    @RequestMapping(value = { "add" }, method = RequestMethod.POST, produces = { MediaType.APPLICATION_JSON_UTF8_VALUE })
+    @RequestMapping(value = {"add"}, method = RequestMethod.POST, produces = {MediaType.APPLICATION_JSON_UTF8_VALUE})
     @ResponseBody
     public RestTriggerResponse addEntity(@RequestBody String body) {
         log.info("---------------------------- Starting Job Submission Validation Add Process From Rest Trigger ----------------------------------------");
 
-	    Map<String, Object> valuesChanges = (Map<String, Object>) convertToMap(body).get("data");
+        Map<String, Object> valuesChanges = (Map<String, Object>) convertToMap(body).get("data");
+
+        RestTriggerRequest<JobSubmission> restTriggerRequest = convertToObject(body);
+
+        Integer entityID = restTriggerRequest.getMeta().getEntityId();
+        Integer updatingUserID = restTriggerRequest.getMeta().getUserId();
+
+
+        String jobOrdersString = jsonConverter.convertEntityToJsonString(valuesChanges.get("jobOrder"));
```

Comment:
Do we need to use the jsonConverter here?  Can't we just .toString() the Object?